scripts/codemods/explicit-barrel-imports.cjs

```diff
@@ -0,0 +1,39 @@
+const fs = require('fs');
+const path = require('path');
+
+module.exports = function (fileInfo, api) {
```

Comment:
Nope, it'll touch ~380 files so I wanted input before going any further. It can either happen in this PR or a future PR. 
In a perfect world my name wouldn't be against all these changes in the git history but I don't think we have the ability to run it via a bot account. 🤔